--- libcomps-libcomps-0.1.6/libcomps/src/CMakeLists.txt.orig	2014-01-29 12:10:07.000000000 +0100
+++ libcomps-libcomps-0.1.6/libcomps/src/CMakeLists.txt	2014-06-15 20:05:25.396247741 +0200
@@ -37,6 +37,7 @@
 add_library(libcomps SHARED ${libcomps_SOURCES})
 target_link_libraries(libcomps ${EXPAT_LIBRARY})
 target_link_libraries(libcomps ${LIBXML2_LIBRARIES})
+target_link_libraries(libcomps m)
 set_target_properties(libcomps PROPERTIES OUTPUT_NAME "comps")
 set_target_properties(libcomps PROPERTIES SOVERSION ${VERSION})
 
