--- libcomps-0.1.24/libcomps/src/python/docs/CMakeLists.txt.orig	2026-02-23 14:44:27.000000000 +0100
+++ libcomps-0.1.24/libcomps/src/python/docs/CMakeLists.txt	2026-05-21 06:23:36.452747941 +0200
@@ -24,8 +24,6 @@ add_custom_target(pydocs)
 add_dependencies(pydocs pycomps)
 
 include(../pycopy.cmake)
-add_custom_command(TARGET pydocs PRE_BUILD COMMAND set -E $ENV{LD_LIBRARY_PATH} "${LIBCOMPS_OUT}:$ENV{LD_LIBRARY_PATH}")
-
 add_custom_command(TARGET pydocs POST_BUILD
                                  COMMAND ${PYTHON_EXECUTABLE} ${SPHINX_EXECUTABLE} -E -b html
                                   "${CMAKE_CURRENT_SOURCE_DIR}/doc-sources/"
