diff -urNp -x '*.orig' libcomps-libcomps-0.1.11.org/libcomps/src/python/docs/CMakeLists.txt libcomps-libcomps-0.1.11/libcomps/src/python/docs/CMakeLists.txt
--- libcomps-libcomps-0.1.11.org/libcomps/src/python/docs/CMakeLists.txt	2019-03-11 08:09:27.000000000 +0100
+++ libcomps-libcomps-0.1.11/libcomps/src/python/docs/CMakeLists.txt	2021-03-07 23:12:47.572339850 +0100
@@ -24,7 +24,6 @@ add_custom_target(pydocs)
 add_dependencies(pydocs pycomps)
 
 include(../pycopy.cmake)
-add_custom_command(TARGET pydocs PRE_BUILD COMMAND set -E $ENV{LD_LIBRARY_PATH} "${LIBCOMPS_OUT}:$ENV{LD_LIBRARY_PATH}")
 
 add_custom_command(TARGET pydocs COMMAND python${pversion} ${SPHINX_EXECUTABLE} -E -b html
                                   "${CMAKE_CURRENT_SOURCE_DIR}/doc-sources/"
