--- libcomps-libcomps-0.1.15/libcomps/src/python/src/CMakeLists.txt~	2020-03-31 11:39:29.000000000 +0200
+++ libcomps-libcomps-0.1.15/libcomps/src/python/src/CMakeLists.txt	2021-03-11 22:02:58.111049060 +0100
@@ -115,8 +115,6 @@
     INSTALL(FILES libcomps/__init__.py DESTINATION libcomps/src/python/src/libcomps)
     INSTALL(TARGETS pycomps LIBRARY DESTINATION libcomps/src/python/src/libcomps)
 ELSE ()
-    EXECUTE_PROCESS(COMMAND ${PYTHON_EXECUTABLE} -c "from sys import stdout; from distutils import sysconfig; stdout.write(sysconfig.get_python_lib(True))" OUTPUT_VARIABLE PYTHON_INSTALL_DIR)
-
     INSTALL(FILES ${pycomps_SRCDIR}/libcomps/__init__.py DESTINATION ${PYTHON_INSTALL_DIR}/libcomps)
     #INSTALL(FILES ${pycomps_SRCDIR}/tests/__test.py DESTINATION
     #              ${PYTHON_INSTALL_DIR}/libcomps/)
